### ISSUE TITLE
Typescript Errors - Namespace 'React' has no exported member 'JSX'

### ISSUE THREAD

**Issue by victsant** [timhagn/gatsby-background-image#143]
Getting the below errors when using typescript linter, I am using the `1.4.1` version of the package.

node_modules/gatsby-background-image/index.d.ts:28:36 - error TS2694: Namespace 'React' has no exported member 'JSX'.

28   type IntrinsicTags = keyof React.JSX.IntrinsicElements;
                                      ~~~

node_modules/gatsby-background-image/index.d.ts:29:52 - error TS2694: Namespace 'React' has no exported member 'JSX'.

29   type DefaultExtraProps = { Tag?: 'div' } & React.JSX.IntrinsicElements['div'];
                                                      ~~~

node_modules/gatsby-background-image/index.d.ts:34:26 - error TS2694: Namespace 'React' has no exported member 'JSX'.

34     : { Tag: U } & React.JSX.IntrinsicElements[U]

**Comment by timhagn** [comment]
Hi @victsant,

please follow the Issue Templates instructions (reproduction + `gatsby info`) or I can't help you - couldn't replicate it with the newest TS + React versions, at least...

Best,

Tim.

**Comment by timhagn** [comment]
Hi @victsant!

Have a look at the just published `gbi@1.5.0` if that solves your TS issue : )!

Best,

Tim.
